pkgs/os-specific/linux/nixos-rebuild/nixos-rebuild.sh

```diff
@@ -501,7 +510,7 @@ else # [ -n "$rollback" ]
         )
         pathToConfig="$profile"-${systemNumber}-link
         if [ -z "$targetHost" ]; then
-            ln -sT "$pathToConfig" ./result
+            ln -sfT "$pathToConfig" ./result
```

Comment:
I found another inconsistency with creating ./result`:
When doing a rollback If with a preexisting link, it causes the error `ln: failed to create symbolic link './result': File exists`. This does not happen on regular rebuilds, Nix just overwrites the file.

Is it fine to include this fix in this PR or should I open a seperate one?

Reply:
Seems fine to me, but maybe do it in a separate commit?

Reply:
The issue with `ln -f` is that it overwrites all types of files, while Nix only overwrites symlinks to make sure it doesn't delete any important data.

I reverted the change, maybe this can be improved in a future PR.